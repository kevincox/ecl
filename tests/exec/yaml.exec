cmd: load json.ecl -fyaml
