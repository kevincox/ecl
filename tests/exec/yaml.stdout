---
bool: false
dict: 
  list: 
    - 1
    - 2
    - 3
  nil_: ~
num: 1
